hr_employee_partner_external/README.rst

```diff
@@ -80,6 +80,10 @@ Contributors
 ------------
 
 -  Stefano Consolaro <stefano.consolaro@mymage.it>
+-  `APSL-Nagarro <https://apsl.tech>`__:
+
+   -  Bernat Obrador <bobrador@apsl.net>
```

Comment:
2 new contributors for this simple migration seems too much.

Reply:
Done.